Demo/Components/BottomSheet/BottomSheetMechanicsDemoViewController.swift

```diff
@@ -246,6 +259,27 @@ extension BottomSheetMechanicsDemoViewController: RootViewControllerDelegate {
 }
 
 extension BottomSheetMechanicsDemoViewController: BottomSheetDelegate {
+    func bottomSheetCanDismiss(_ bottomSheet: BottomSheet) -> Bool {
```

Comment:
Maybe change this to `bottomSheetShouldDismiss`? I think that is Apple's convention 🍎